docs/config-file/v2.rst

```diff
@@ -298,6 +298,9 @@ Configuration for the documentation build process.
 
    build:
      image: latest
+     apt_packages:
+       - mysql-client
+       - cmatrix
```

Comment:
I'd suggest to find _useful_ packages used to build docs to use as the main example in our documentation.

Reply:
Yeah, wasn't sure what are the  common packages that people require. I think they are mostly very specific packages for some deps.